spec/round_spec.rb

```diff
@@ -0,0 +1,44 @@
+require './lib/deck'
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
```

Comment:
It looks like you're getting an error for this spec:
`NameError: uninitialized constant Round`
If you update line 4 here to be `require './lib/round'` it should fix that error.